src/System.Net.NetworkInformation/tests/FunctionalTests/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "System.Net.Primitives": "4.0.10-beta-*",
+    "System.Net.Sockets": "4.1.0-beta-23225",
```

Comment:
See above why we can't still use contracts: Sockets will flow from the wrong implementation for Windows.